aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/UnifiedPushMessage.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.aerogear.android.unifiedpush.gcm;
+
+public interface UnifiedPushMessage {
+
+    /**
+     * Constant for the "push message" key
+     */
+    String MESSAGE = "alert";
```

Comment:
Nevermind: https://github.com/danielpassos/unified-push-helloworld/commit/aaa14c4808b020c54905c5e79215ddcfede3f113#commitcomment-11235908

However, I'd suggest to name it `ALERT_KEY` and using this JavaDoc:

```
/**
 * Constant for the name of the push message alert key
 */
```

Because the message has more Keys, not just alert